packages/blocks/src/root-block/edgeless/components/block-portal/note/edgeless-note.ts

```diff
@@ -188,6 +189,49 @@ export class EdgelessBlockPortalNote extends EdgelessPortalBase<NoteBlockModel>
     const x = clamp(e.x, rect.left + offsetX, rect.right - offsetX);
     const y = clamp(e.y, rect.top + offsetY, rect.bottom - offsetY);
     handleNativeRangeAtPoint(x, y);
+
+    // add an empty paragraph
```

Comment:
In this case, when the last block is a non-text block, a paragraph block will always be added at the bottom. The logic here should be to focus on the nearest text block first when clicking. If the nearest block is not text block, and the last block is not a text block, then add an empty block